README.md

```diff
@@ -53,6 +53,20 @@ On Linux, you may install the .deb file, then run `/opt/rcv/bin/RCTab` to launch
     
     `$ chmod 777 gradlew`
 
+#### Method 3 (Least Easy): Building on an airgapped Machine
+
+
+1. Download gradle from https://gradle.org/releases/ and place it in your path
+2. Download and extract the source code from [releases page](https://github.com/BrightSpots/rcv/releases)
+3. Download the appropriate cache files for your OS: cache.[OS].zip
+4. Stop the gradle daemon with `gradle --stop`
+5. Delete the directory ~/.gradle/caches if it exists
+6. Extract the appropriate caches/[filename].zip to ~/.gradle/caches so that the "caches" directory is in ~/.gradle
+7. In the extracted directory, you may manually verify each dependency using checksums.csv in accordance with your own policies
+8. Run `gradle assemble --offline` and ensure you get no errors
+9. Run `gradle run --offline` to launch RCTab
+10. Run `gradle jpackage --offline` to generate a jpackage package
```

Comment:
Are all these meant to be run in sequence? It seems like maybe you could at least combine the last two saying "Run X to XX or Y to YY".

Also, will people know what "a jpackage package" is?